notes/programming-languages/python/packages/nltk.md

```diff
@@ -0,0 +1,76 @@
+# Python Language Overview
+
+### The `nltk` Module
```

Comment:
I'd prefer to use the term "Package" to refer to third-party open source packages downloadable via pip, and reserve the term "Module" for built-in Python modules which don't require additional installation.